waspc/src/Wasp/Generator/SdkGenerator.hs

```diff
@@ -80,9 +80,7 @@ genSdkReal spec =
     [ genFileCopy [relfile|api/index.ts|],
       genFileCopy [relfile|api/events.ts|],
       genFileCopy [relfile|core/config.ts|],
-      genFileCopy [relfile|core/auth.ts|],
```

Comment:
This file was generated (but weren't used) even when the user didn't define auth. However, the `auth` folder isn't generated under the same conditions.

Therefore, imports in `core/auth.ts` that refer to something from `wasp/auth` would cause build problems in apps that don't use auth.